Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to goerli
+   a) support restarting of nodes
+3. Improvements to Management contract
+   a) Support withdrawals
+4. Mempool improvements
+   a) Remove failed transactions
+
+
+## V3
+1. Support a limited number of user operated nodes
+   a) Catching up robustness
+   b) Script robustness
+   c) Instructions
+   d) Try outside of Azure?
+2. Design and implement reveleation period
+3. Finalise cryptography
+   a) key derivation
```

Comment:
```suggestion
   a) network key derivation
```

Reply:
there are multiple derivations that are needed. 
I preferred to use a more general term

Reply:
I was under the impression it was network key derivation that happens on each rollup. 
Ooc, what are the other ones ?

Reply:
https://whitepaper.obscu.ro/obscuro-whitepaper/detailed-design.html#generating-keys